nixos/modules/services/misc/autosuspend.nix

```diff
@@ -0,0 +1,238 @@
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}:
+with lib; let
```

Comment:
Yeah I have `alejandra` as a default nix formatter, but I should disable it when I contribute to nixpkgs.  I ran `nixpkgs-fmt` on everything and fixed a few things :-)

Reply:
As for `with lib`, I agree that it might be better without it, for the sake of explicitness.  Would be great to have an action in my language server to automatically import it, doing this manually is a chore :sweat:

Reply:
Eh, I still went ahead and did it :sweat_smile: didn't take much time in the end.